tests/tests/swfs/avm2/pixelbender_dithering/test.toml

```diff
@@ -0,0 +1,10 @@
+# Based on  https://github.com/FlashBacks1998/Custom-Pixel-Bender-Example-Ordered-Dithering-
+# The prints were sorted to avoid relying on object iteration order
+num_ticks = 1
+
+[image_comparisons.output]
+tolerance = 1
+max_outliers = 19813
```

Comment:
That's... um... a lot?
Perhaps a bit higher tolerance could be considered instead?